client/core/trade.go

```diff
@@ -186,7 +186,7 @@ func (t *trackedTrade) coreOrderInternal() *Order {
 // BUY order.
 // lockedAmount should be called with the mtx >= RLocked.
 func (t *trackedTrade) lockedAmount() (locked uint64) {
-	if t.coinsLocked { // implies no swap has been sent
+	if t.coinsLocked { // implies no swap has been sent or the trade has been resumed on restart after swap
```

Comment:
Isn't this not true anymore because of your work here? Only pre-swap startup loads will have coins locked, right?

Reply:
`coinsLocked==true` means one of two things currently:
- swap has NOT yet been sent, and this is normal negotiation with no restart
- swap has been sent, but there was a restart that put change coin in `coins` and set `coinsLocked=true` (instead of `change` where it would go in the first bullet)

Reply:
I revised this comment.  Let me know if it makes more sense now.
BTW, I'd like to test a resumed trade for a partially filled order that is still booked.

Reply:
I see now. We're almost certainly handling this wrong at startup, but let's look at that part separately.

Reply:
Right I am hesitant to change it to start assigning to change/changeLocked, but that would be closer to restoring things properly.